test/fixtures/dumpio.js

```diff
@@ -0,0 +1,9 @@
+(async() => {
+  const [, , puppeteerRoot, options, emptyPage] = process.argv;
+  const browser = await require(puppeteerRoot).launch(JSON.parse(options));
+  const page = await browser.newPage();
+  await page.goto(emptyPage);
+  await page.evaluate(() => console.log('test'));
```

Comment:
nit: let's log `"MAGIC_DUMPIO_TEST"` to be distinct

Reply:
Sure, done...